scripts/sh/destroy_local_vms.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+declare -a local_vms=(\
+    "wgclient-ubuntu-bionic-x86_64" \
+    "jenkins_master-centos-7.5-x86_64" \
+    "rust_slave-centos-7.5-x86_64" \
+    "docker_slave-centos-7.5-x86_64" \
+    "rust_slave-ubuntu-trusty-x86_64" \
+    "base-windows-2012_r2-x86_64" \
+    "rust_slave_git_bash-windows-2012_r2-x86_64" \
+    "jenkins_rust_slave-windows-2016-x86_64" \
+    "rust_slave_msys2-windows-2012_r2-x86_64")
```

Comment:
Missed `travis_rust_slave-windows-2016-x86_64` from this list

Reply:
Thanks for this. Pushed another commit, and it also contains the documentation change.